lang/java/reef-examples/src/main/java/org/apache/reef/examples/group/bgd/BGDClient.java

```diff
@@ -67,7 +67,6 @@ public BGDClient(@Parameter(InputDir.class) final String input,
    *
    * @param runtimeConfiguration the runtime to run on.
    * @param jobName              the name of the job on the runtime.
-   * @return
```

Comment:
Why do you sometimes add empty return tag (BuilderUtils.java), and sometimes remove it? Shouldn't there be one approach?